pkg/machine/shim/host_test.go

```diff
@@ -0,0 +1,75 @@
+package shim
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_validateDestinationPaths(t *testing.T) {
+	type args struct {
+		dest string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name:    "Expect fail - /tmp",
+			args:    args{"/tmp"},
```

Comment:
This is inconsistent with the rest, overall I see no need for the args struct, might as well just inline the dest filed into the tests struct.